### PR TITLE
Updated checkstyle file to match formatter and cleanup profile

### PR DESCRIPTION
**Pull request by MMaiero**
First step for this feature.

Signed-off-by: Maiero <matteo.maiero@eurotech.com>

Brief description of the PR. This PR changes the checkstyle file. The checks will be performed during PRs build inn order to maintain a certain code style through all the project.

### CODE REVIEW

**Comment by MMaiero** [comment]
Some projects are ignored because they require a more extensive work that will be performed in upcoming PRs.

File: kura/emulator/org.eclipse.kura.emulator.clock/src/main/java/org/eclipse/kura/emulator/clock/ClockServiceImpl.java [hunk 1]
```diff
@@ -24,14 +24,14 @@
 
 public class ClockServiceImpl implements ConfigurableComponent, ClockService {
 
-    private static final Logger s_logger = LoggerFactory.getLogger(ClockServiceImpl.class);
+    private static final Logger logger = LoggerFactory.getLogger(ClockServiceImpl.class);
```
Comment (pierantoniomerlino) [inline kura/emulator/org.eclipse.kura.emulator.clock/src/main/java/org/eclipse/kura/emulator/clock/ClockServiceImpl.java:27]: Update copyright.

File: kura/emulator/org.eclipse.kura.emulator.gpio/src/main/java/org/eclipse/kura/emulator/gpio/EmulatedPin.java [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 public class EmulatedPin implements KuraGPIOPin {
 
-    private static final Logger s_logger = LoggerFactory.getLogger(EmulatedPin.class);
+    private static final Logger logger = LoggerFactory.getLogger(EmulatedPin.class);
```
Comment (pierantoniomerlino) [inline kura/emulator/org.eclipse.kura.emulator.gpio/src/main/java/org/eclipse/kura/emulator/gpio/EmulatedPin.java:29]: Update copyright

File: kura/emulator/org.eclipse.kura.emulator.net/src/main/java/org/eclipse/kura/emulator/net/EmulatedNetworkServiceImpl.java [hunk 3]
```diff
@@ -36,13 +36,13 @@
 
 public class EmulatedNetworkServiceImpl implements NetworkService {
 
-    private static final Logger s_logger = LoggerFactory.getLogger(EmulatedNetworkServiceImpl.class);
+    private static final Logger logger = LoggerFactory.getLogger(EmulatedNetworkServiceImpl.class);
```
Comment (pierantoniomerlino) [inline kura/emulator/org.eclipse.kura.emulator.net/src/main/java/org/eclipse/kura/emulator/net/EmulatedNetworkServiceImpl.java:39]: Update copyright

File: kura/emulator/org.eclipse.kura.emulator.watchdog/src/main/java/org/eclipse/kura/emulator/watchdog/WatchdogServiceImpl.java [hunk 4]
```diff
@@ -39,8 +39,8 @@
     private Map<String, Object> properties;
     private ScheduledExecutorService executor;
     private ScheduledFuture<?> future;
-    private int pingInterval = 10000;	// milliseconds
-    private boolean configEnabled = false;	// initialized in properties, if false -> no watchdog
+    private int pingInterval = 10000; // milliseconds
```
Comment (pierantoniomerlino) [inline kura/emulator/org.eclipse.kura.emulator.watchdog/src/main/java/org/eclipse/kura/emulator/watchdog/WatchdogServiceImpl.java:42]: Update copyright

File: kura/emulator/org.eclipse.kura.emulator/src/main/java/org/eclipse/kura/emulator/Emulator.java [hunk 5]
```diff
@@ -30,10 +30,10 @@
 
     private static final String SNAPSHOT_0_NAME = "snapshot_0.xml";
```
Comment (pierantoniomerlino) [inline kura/emulator/org.eclipse.kura.emulator/src/main/java/org/eclipse/kura/emulator/Emulator.java:32]: Update copyright

File: kura/org.eclipse.kura.api/pom.xml [hunk 6]
```diff
@@ -92,6 +92,13 @@
                     </base>
                 </configuration>
             </plugin>
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/pom.xml:94]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/KuraException.java [hunk 7]
```diff
@@ -14,9 +14,9 @@
 import java.util.MissingResourceException;
 import java.util.ResourceBundle;
 
-import org.osgi.annotation.versioning.ProviderType;
-import org.apache.logging.log4j.Logger;
 import org.apache.logging.log4j.LogManager;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/KuraException.java:17]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/KuraInvalidMetricTypeException.java [hunk 8]
```diff
@@ -17,15 +17,15 @@
  * @noextend This class is not intended to be subclassed by clients.
  */
 @ProviderType
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/KuraInvalidMetricTypeException.java:19]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/KuraRuntimeException.java [hunk 9]
```diff
@@ -14,9 +14,9 @@
 import java.util.MissingResourceException;
 import java.util.ResourceBundle;
 
-import org.osgi.annotation.versioning.ProviderType;
-import org.apache.logging.log4j.Logger;
 import org.apache.logging.log4j.LogManager;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/KuraRuntimeException.java:17]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/bluetooth/le/beacon/BluetoothLeBeacon.java [hunk 10]
```diff
@@ -40,55 +40,55 @@ public BluetoothLeBeacon() {
     }
 
     public boolean isLeBrHost() {
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/bluetooth/le/beacon/BluetoothLeBeacon.java:42]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/bluetooth/listener/BluetoothAdvertisementData.java [hunk 11]
```diff
@@ -18,9 +18,9 @@
 
 /**
  * @noextend This class is not intended to be subclassed by clients.
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/bluetooth/listener/BluetoothAdvertisementData.java:20]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/channel/Channel.java [hunk 12]
```diff
@@ -119,13 +119,13 @@ public DataType getValueType() {
 
     /**
      * Returns a boolean indicating if this channel is enabled or not
-     * 
+     *
      * @since 1.4
      * @return a boolean indicating if this channel is enabled or not
      */
 
     public boolean isEnabled() {
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/channel/Channel.java:127]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/channel/ChannelRecord.java [hunk 13]
```diff
@@ -27,24 +27,24 @@
  * The Class ChannelRecord contains the information needed for performing a read or write
  * operation on a specific channel.<br/>
  * <br/>
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/channel/ChannelRecord.java:29]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/channel/ChannelStatus.java [hunk 14]
```diff
@@ -143,15 +143,15 @@ public String getExceptionMessage() {
     public int hashCode() {
         final int prime = 31;
         int result = 1;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/channel/ChannelStatus.java:145]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/cloud/CloudService.java [hunk 15]
```diff
@@ -12,6 +12,7 @@
 package org.eclipse.kura.cloud;
 
 import org.eclipse.kura.KuraException;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/cloud/CloudService.java:14]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/cloud/factory/CloudServiceFactory.java [hunk 16]
```diff
@@ -17,8 +17,7 @@
 
 import org.eclipse.kura.KuraException;
 import org.eclipse.kura.cloud.CloudService;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/cloud/factory/CloudServiceFactory.java:19]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/deployment/hook/RequestContext.java [hunk 17]
```diff
@@ -14,18 +14,18 @@
 
 /**
  * This class provides some context information describing a DEPLOY-V2 request.
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/deployment/hook/RequestContext.java:16]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/NetworkState.java [hunk 18]
```diff
@@ -16,14 +16,20 @@
...
+    /** A network device is connecting to a network and there is no other available network connection. */
+    CONNECTING(40),
+    /** A network device is connected, but there is only link-local connectivity. */
+    CONNECTED_LOCAL(50),
+    /** A network device is connected, but there is only site-local connectivity. */
+    CONNECTED_SITE(60),
+    /** A network device is connected, with global network connectivity. */
     CONNECTED_GLOBAL(70);
 
     private int m_code;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/NetworkState.java:35]: m_code -> code

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/modem/ModemConfig.java [hunk 19]
```diff
@@ -180,6 +180,7 @@ public void setUsername(String username) {
      * @return password as {@link String}
      * @deprecated
      */
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/modem/ModemConfig.java:182]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/modem/ModemPdpContext.java [hunk 20]
```diff
@@ -16,9 +16,9 @@
  */
 public class ModemPdpContext {
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/modem/ModemPdpContext.java:18]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/modem/SerialModemDevice.java [hunk 21]
```diff
@@ -50,36 +50,45 @@ public String getManufacturerName() {
     public int hashCode() {
         final int prime = 31;
         int result = 1;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/modem/SerialModemDevice.java:52]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/position/NmeaPosition.java [hunk 22]
```diff
@@ -40,7 +40,7 @@
  * <li>latitudeHemisphere : hemisphere of the latitude = N or S
  * <li>longitudeHemisphere : hemisphere of the longitude = E or W
  * </ul>
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/position/NmeaPosition.java:42]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/BooleanValue.java [hunk 23]
```diff
@@ -51,7 +51,7 @@ public BooleanValue(final boolean value) {
     @Override
     public int compareTo(final TypedValue<Boolean> otherTypedValue) {
         requireNonNull(otherTypedValue, "Typed Value cannot be null");
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/BooleanValue.java:53]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/ByteArrayValue.java [hunk 24]
```diff
@@ -57,7 +57,7 @@ public ByteArrayValue(final byte[] value) {
     public int compareTo(final TypedValue<byte[]> otherTypedValue) {
         requireNonNull(otherTypedValue, "Typed Value cannot be null");
         final byte[] otherValue = otherTypedValue.getValue();
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/ByteArrayValue.java:59]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/DoubleValue.java [hunk 25]
```diff
@@ -92,7 +92,7 @@ public int hashCode() {
         int result = 1;
         long temp;
         temp = Double.doubleToLongBits(this.value);
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/DoubleValue.java:94]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/FloatValue.java [hunk 26]
```diff
@@ -66,7 +66,7 @@ public boolean equals(Object obj) {
             return false;
         }
         FloatValue other = (FloatValue) obj;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/FloatValue.java:68]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/IntegerValue.java [hunk 27]
```diff
@@ -90,7 +90,7 @@ public Integer getValue() {
     public int hashCode() {
         final int prime = 31;
         int result = 1;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/IntegerValue.java:92]: Update copyright
+

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/StringValue.java [hunk 28]
```diff
@@ -95,7 +95,7 @@ public String getValue() {
     public int hashCode() {
         final int prime = 31;
         int result = 1;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/type/StringValue.java:97]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/wire/WireConfiguration.java [hunk 29]
```diff
@@ -161,8 +161,8 @@ public boolean equals(final Object obj) {
     public int hashCode() {
         final int prime = 31;
         int result = 1;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/wire/WireConfiguration.java:163]: Update copyright

File: kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/wire/graph/MultiportWireConfiguration.java [hunk 30]
```diff
@@ -52,24 +52,29 @@ public void setReceiverPort(int receiverPort) {
     public int hashCode() {
         final int prime = 31;
         int result = super.hashCode();
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/wire/graph/MultiportWireConfiguration.java:54]: Update copyright

File: kura/org.eclipse.kura.asset.provider/src/main/java/org/eclipse/kura/internal/asset/provider/BaseAssetConfiguration.java [hunk 31]
```diff
@@ -58,8 +58,8 @@
     private Map<String, Object> properties;
     private Tocd ocd;
     private AssetConfiguration assetConfiguration;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.asset.provider/src/main/java/org/eclipse/kura/internal/asset/provider/BaseAssetConfiguration.java:60]: Update copyright

File: kura/org.eclipse.kura.ble.provider/src/main/java/org/eclipse/kura/internal/ble/BluetoothLeServiceImpl.java [hunk 32]
```diff
@@ -115,13 +115,13 @@ private void startBluetoothDaemon() {
     }
 
     private boolean startBluetoothUbuntuSnap() {
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.ble.provider/src/main/java/org/eclipse/kura/internal/ble/BluetoothLeServiceImpl.java:117]: Update copyright

File: kura/org.eclipse.kura.broker.artemis.core/src/main/java/org/eclipse/kura/broker/artemis/core/ServerRunner.java [hunk 33]
```diff
@@ -5,7 +5,7 @@
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.broker.artemis.core/src/main/java/org/eclipse/kura/broker/artemis/core/ServerRunner.java:7]: Update copyright

File: kura/org.eclipse.kura.broker.artemis.core/src/main/java/org/eclipse/kura/broker/artemis/core/UserAuthentication.java [hunk 34]
```diff
@@ -22,9 +22,9 @@
 
     public static class User {
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.broker.artemis.core/src/main/java/org/eclipse/kura/broker/artemis/core/UserAuthentication.java:24]: Update copyright

File: kura/org.eclipse.kura.camel.xml/src/org/eclipse/kura/camel/xml/XmlRouterComponent.java [hunk 35]
```diff
@@ -72,18 +72,18 @@ protected void customizeBuilder(final Builder builder, final Map<String, Object>
 
         // JMX
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.camel.xml/src/org/eclipse/kura/camel/xml/XmlRouterComponent.java:74]: Update copyright

File: kura/org.eclipse.kura.camel/src/main/java/org/eclipse/kura/camel/runner/CamelRunner.java [hunk 36]
```diff
@@ -64,13 +64,7 @@
     public static ContextFactory createOsgiFactory(final BundleContext bundleContext) {
         Objects.requireNonNull(bundleContext);
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.camel/src/main/java/org/eclipse/kura/camel/runner/CamelRunner.java:66]: Update copyright

File: kura/org.eclipse.kura.cloudconnection.raw.mqtt.provider/src/main/java/org/eclipse/kura/cloudconnection/raw/mqtt/cloud/RawMqttCloudEndpoint.java [hunk 37]
```diff
@@ -99,7 +99,7 @@ public void deactivated() {
         this.dataService.removeDataServiceListener(this);
 
         synchronized (this) {
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.cloudconnection.raw.mqtt.provider/src/main/java/org/eclipse/kura/cloudconnection/raw/mqtt/cloud/RawMqttCloudEndpoint.java:101]: Update copyright

File: kura/org.eclipse.kura.cloudconnection.raw.mqtt.provider/src/main/java/org/eclipse/kura/cloudconnecton/raw/mqtt/util/AbstractStackComponent.java [hunk 38]
```diff
@@ -77,17 +77,17 @@ public void deactivated() {
     }
 
     protected StackComponentOptions<T> getOptions() {
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.cloudconnection.raw.mqtt.provider/src/main/java/org/eclipse/kura/cloudconnecton/raw/mqtt/util/AbstractStackComponent.java:79]: Update copyright

File: kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ComponentMetaTypeBundleTracker.java [hunk 39]
```diff
@@ -63,7 +63,7 @@ public Bundle addingBundle(Bundle bundle, BundleEvent event) {
     @Override
     public void removedBundle(Bundle bundle, BundleEvent event, Bundle object) {
         super.removedBundle(bundle, event, object);
-        m_configurationService.onBundleRemoved(bundle);
+        this.m_configurationService.onBundleRemoved(bundle);
     }
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ComponentMetaTypeBundleTracker.java:67]: m_configurationService -> configurationService

File: kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/data/store/HouseKeeperTask.java [hunk 40]
```diff
@@ -25,7 +25,7 @@
     private static final Logger s_logger = LoggerFactory.getLogger(HouseKeeperTask.class);
 
     private final int m_purgeAge;
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/data/store/HouseKeeperTask.java:27]: m_purgeAge -> purgeAge

File: kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/data/store/HouseKeeperTask.java [hunk 41]
```diff
@@ -42,7 +42,7 @@ public void run() {
 
             //
             // check and attempt to repair the store
-            if (doRepair) {
+            if (this.doRepair) {
                 s_logger.info("HouseKeeperTask: Check store...");
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/data/store/HouseKeeperTask.java:46]: s_logger -> logger

File: kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/util/ProcessUtil.java [hunk 42]
```diff
@@ -42,15 +41,11 @@ public static SafeProcess exec(String command) throws IOException {
...
-        Future<SafeProcess> futureSafeProcess = s_processExecutor.submit(new Callable<SafeProcess>() {
-
-            @Override
-            public SafeProcess call() throws Exception {
-                Thread.currentThread().setName("SafeProcessExecutor");
-                SafeProcess safeProcess = new SafeProcess();
-                safeProcess.exec(cmdarray);
-                return safeProcess;
-            }
+        Future<SafeProcess> futureSafeProcess = s_processExecutor.submit(() -> {
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.core/src/main/java/org/eclipse/kura/core/util/ProcessUtil.java:44]: s_processExecutor -> processExecutor

File: kura/org.eclipse.kura.driver.s7plc.provider/src/main/java/org/eclipse/kura/internal/driver/s7plc/S7PlcDriver.java [hunk 43]
```diff
@@ -216,6 +216,10 @@ public synchronized void read(int db, int offset, byte[] data) throws IOExceptio
     @SuppressWarnings("serial")
     static final class Moka7Exception extends IOException {
 
+        /**
+         *
+         */
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.driver.s7plc.provider/src/main/java/org/eclipse/kura/internal/driver/s7plc/S7PlcDriver.java:221]: Remove empty comment block

File: kura/org.eclipse.kura.linux.debian.provider/src/main/java/org/eclipse/kura/internal/linux/net/config/NetInterfaceConfigSerializationServiceImpl.java [hunk 44]
```diff
@@ -221,10 +222,12 @@ private Properties parseDebianConfigFile(File ifcfgFile, String interfaceName) t
...
-                            // once the correct interface is found, read all
-                            // configuration information
-                            else if ("iface".equals(args[0]) && args[1].equals(interfaceName)) {
+                            } else if ("iface".equals(args[0]) && args[1].equals(interfaceName)) { // once the correct
+                                                                                                   // interface is
+                                                                                                   // found,
+                                                                                                   // read all
+                                                                                                   // configuration
+                                                                                                   // information
                                 kuraProps.setProperty(BOOTPROTO_PROP_NAME, args[3]);
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.linux.debian.provider/src/main/java/org/eclipse/kura/internal/linux/net/config/NetInterfaceConfigSerializationServiceImpl.java:231]: It isn't a good intentation...

File: kura/org.eclipse.kura.protocol.can/src/main/java/org/eclipse/kura/protocol/can/CanMessage.java [hunk 45]
```diff
@@ -13,23 +13,23 @@
 
 public class CanMessage {
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.protocol.can/src/main/java/org/eclipse/kura/protocol/can/CanMessage.java:15]: Update copyright

File: kura/org.eclipse.kura.rest.provider/src/main/java/org/eclipse/kura/internal/rest/provider/RestServiceOptions.java [hunk 46]
```diff
@@ -7,19 +7,18 @@
  * http://www.eclipse.org/legal/epl-v10.html
  *
  *******************************************************************************/
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.rest.provider/src/main/java/org/eclipse/kura/internal/rest/provider/RestServiceOptions.java:9]: Update copyright

File: kura/org.eclipse.kura.stress/src/main/java/org/eclipse/kura/stress/Stress.java [hunk 47]
```diff
@@ -26,7 +26,7 @@
 
 public class Stress implements ConfigurableComponent {
```
Comment (pierantoniomerlino) [inline kura/org.eclipse.kura.stress/src/main/java/org/eclipse/kura/stress/Stress.java:28]: Update copyright

**Review by pierantoniomerlino — APPROVED** [review]

**Pull request merged**